ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -200,26 +249,103 @@ def _parse_table_columns(self, table_response, tbl_entity, name) -> List[Column]
 
     def get_database_entity(self, database_name: str) -> Database:
         return Database(
+            id=uuid.uuid4(),
             name=database_name,
-            service=EntityReference(id=self.service.id, type=self.config.service_type),
+            service=EntityReference(
+                id=self.service.id, type=self.service_connection.serviceType
+            ),
+        )
+
+    def create_database_service(self, dbService):
```

Comment:
We don't won't to create the service right? I think we can remove this and line 116
```
yield from self.create_database_service(self.service_connection.dbService)
```

Reply:
Hi @pmbrull we are calling `self.metadata.get_create_service_from_source` in that method, we are not calling any backend api to create service. We are doing here same as we have done in Amundsen

Reply:
aaaah I see! Then this is just me being picky, but could we please update the name of the function? Something like `get_database_service` and `get_messaging_service` to be more clear about that?

Also I am seeing that we do `yield create_service_entity`. This might not be needed as the service we are sure it will be there right? I am assuming that just a `get_by_name` should be enough to store the service info.

@ulixius9 this we might need to clean on Amundsen as well